packages/flutter/lib/src/widgets/text.dart

```diff
@@ -126,9 +126,14 @@ class DefaultTextStyle extends InheritedTheme {
   /// Whether the text should break at soft line breaks.
   ///
   /// If false, the glyphs in the text will be positioned as if there was unlimited horizontal space.
+  /// 
+  /// This also decides basis of [overflow] property.
   final bool softWrap;
 
   /// How visual overflow should be handled.
+  /// 
+  /// This will be applied on a word basis when [softWrap] is true, and on an alphabet basis when
```

Comment:
What is "word base" or "alphabet base"? I think we'd need to explain this a little better.